starknet_py/contract.py

```diff
@@ -370,9 +376,17 @@ def functions(self) -> FunctionsRepository:
     def address(self) -> int:
         return self._data.address
 
+    @property
+    def data(self) -> ContractData:
```

Comment:
no need to write properties like this, you can do just `self.data = ...` and `self.client = ...` if you want this field to be "public"